tests/test_install.ts

```diff
@@ -230,11 +230,18 @@ describe("Agama test", function () {
             await page.locator("button::-p-text('Continue')").click();
 
             // Check if installation procedure is progressing
-            let progressTitleInstall = await page
-                .locator("#progress-title")
-                .map(header => (header as HTMLElement).innerText)
-                .wait();
-            expect(progressTitleInstall).to.eql("Installing the system, please wait ...");
+            // Sometimes Agama Web interface don't update fast enough the #progress-title value
+            // This lead to random errors of its previous value is not the same than requested.
+            // We added a for loop that checks 3 times if the #progress-title has the required innerText.
+            let installation_started = false;
+            for (let i = 0; i < 3 && !installation_started; i++) {
```

Comment:
this looks very hacky, if transition screen are not not easy to catch, it is better not to check them.

Reply:
I tackled this in https://github.com/jknphy/agama-integration-test-webpack the problem is well know in UI I think, that you might see a screen for a moment, then miliseconds later changes and then come back to the original, this happens in YaST for example that we see initialization values for fraction of a second. This kind of thing can be filed as bug but in general we don't put much effort on that other than the own will of the contributor and in a manual testing session, for is safer to just search for what you are looking for from the beginning, instead of having two steps to find it and to assert its value.